src/sfizz/Synth.cpp

```diff
@@ -892,6 +895,12 @@ void sfz::Synth::hdcc(int delay, int ccNumber, float normValue) noexcept
         return;
     }
 
+    if (ccNumber == config::allNotesOffCC || ccNumber == config::allSoundOffCC) {
```

Comment:
Why? registerCC already does this below.
(also `all sound off` gets itself processed out of order with the midi stream, but that's something else)

Reply:
The point was to not let the voice reset itself.